mycli/main.py

```diff
@@ -668,11 +719,20 @@ def cli(database, user, host, port, socket, password, dbname,
     # Choose which ever one has a valid value.
     database = database or dbname
 
+    ssl = {
+        'ca': ssl_ca,
+        'capath': ssl_capath,
+        'cert': ssl_cert,
+        'key': ssl_key,
+        'cipher': ssl_cipher,
```

Comment:
Can you add os.path.expanduser() to all the file paths here? 

Otherwise specifying `~/server-ca.pem` will fail.